src/js/components/award/shared/activity/ContractGrantActivityChart.jsx

```diff
@@ -303,10 +303,13 @@ const ContractGrantsActivityChart = ({
     ]);
     // sets the line values - hook - runs on mount and dates change
     useEffect(() => {
-        setStartLineData(Object.assign({}, startLineData, { value: lineHelper(dates._startDate) }));
-        setEndLineData(Object.assign({}, endLineData, { value: lineHelper(dates._endDate) }));
-        setPotentialEndLineData(Object.assign({}, potentialEndLineData, { value: lineHelper(dates._potentialEndDate) }));
-    }, [dates]);
+        if (xDomain?.length) {
```

Comment:
@jonhill13 

I dont think this is a good way to do a null check. I could be wrong, but the bug with the tooltip in the component library was fixed by using an `Object.keys(obj).includes()` rather than this kinda thig.

Reply:
xDomain is an array

Reply:
so if I am reading this correctly if xDomain does not exist it will be false and if it exists and there is a length it will be true?

Reply:
I'm just saying this same logic was the root cause of a bug elsewhere. Only difference is its the property of an array rather than an object.

Reply:
so what do you want this to be? `if (xDomain && xDomain.length)`?

Reply:
```suggestion
if (xDomain && xDomain.length > 0)
```